apple/cmake/MakeiOSAppBundleAndLinkFrameworks.cmake

```diff
@@ -11,6 +9,7 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
   MACOSX_BUNDLE_GUI_IDENTIFIER "com.sourcehold.\${PRODUCT_NAME:identifier}"
   MACOSX_BUNDLE_SHORT_VERSION_STRING ${PROJECT_VERSION}
   MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION}
+  XCODE_ATTRIBUTE_ONLY_ACTIVE_ARCH NO
```

Comment:
Replace set(CMAKE_XCODE_ATTRIBUTE_ONLY_ACTIVE_ARCH NO) with XCODE_ATTRIBUTE_ONLY_ACTIVE_ARCH target property, because CMake for some reason ignores global vars during Xcode project generation.